models/Program.ts

```diff
@@ -0,0 +1,31 @@
+// TODO: weekday field got moved to class so we can move this to Class.ts after
+// export enum weekdays {
+//     MON = "MON",
+//     TUE = "TUE",
+//     WED = "WED",
+//     THU = "THU",
+//     FRI = "FRI",
+//     SAT = "SAT",
+//     SUN = "SUN",
+// }
+
+/**
+ * Enum for the program format
+ */
+export enum programFormat {
+    online = "online",
+    inPerson = "in_person",
+    blended = "blended",
+}
+
+/**
+ * Custom type for creating a program
```

Comment:
can you make this more descriptive - maybe something like `Request Body Input for POST /program`